kybra/compiler/install_rust_dependencies.sh

```diff
@@ -47,24 +48,32 @@ function update_rustup() {
 }
 
 function install_wasm32() {
-    echo -e "2/4) Installing wasm32"
+    echo -e "2/5) Installing wasm32"
 
     "$global_kybra_rustup_bin" target add wasm32-wasi &> "$global_kybra_logs_dir"/install_wasm32_wasi
-    "$global_kybra_rustup_bin" target add wasm32-unknown-unknown &> "$global_kybra_logs_dir"/install_wasm32_unknown_unknown
 }
 
 function install_wasi2ic() {
-    echo -e "3/4) Installing wasi2ic"
+    echo -e "3/5) Installing wasi2ic"
 
     "$global_kybra_cargo_bin" install --git https://github.com/wasm-forge/wasi2ic --rev 7418e0bd1a7810c8e9c55cc0155c921503a793b8 &> "$global_kybra_logs_dir"/install_wasi2ic
 }
 
-function install_ic_wasm() {
-    echo -e "4/4) Installing ic-wasm"
+function install_rust_python() {
+    echo -e "4/5) Installing RustPython"
 
-    if [ "$1" -eq 1 ]; then
-        "$global_kybra_cargo_bin" install ic-wasm --version 0.3.6 &> "$global_kybra_logs_dir"/install_ic_wasm
-    fi
+    cd "$global_kybra_version_dir" &> "$global_kybra_logs_dir"/install_rust_python
```

Comment:
I'm not sure exactly which part you're referring to, we can discuss more in a call if you'd like.

Before we stored RustPython in the rust/rust_version (IIRC) directory. Now we store it in the kybra_version directory, so that the dev gets a specific version of RustPython for every version of Kybra. This makes it easy to ensure that the canister always compiles the correct stdlib from the correct RustPython location. It is theoretically possible to update RustPython without updating the Rust version, so I think this is better/correct.